var/spack/repos/builtin/packages/py-aiofiles/package.py

```diff
@@ -13,7 +13,10 @@ class PyAiofiles(PythonPackage):
     homepage = "https://github.com/Tinche/aiofiles"
     pypi = "aiofiles/aiofiles-0.5.0.tar.gz"
 
+    version("0.7.0", sha256="a1c4fc9b2ff81568c83e21392a82f344ea9d23da906e4f6a52662764545e19d4")
     version("0.5.0", sha256="98e6bcfd1b50f97db4980e182ddd509b7cc35909e903a8fe50d8849e02d815af")
 
-    depends_on("py-setuptools", type="build")
-    depends_on("py-wheel", type="build")
```

Comment:
Also needs:
```python
depends_on("python@3.6:3", when="@0.7:", type=("build", "run"))
```
This will prevent it from using Python 4 if it ever gets released